scilpy/io/streamlines.py

```diff
@@ -144,3 +146,52 @@ def load_tractogram_with_reference(parser, args, filepath,
         parser.error('{} is an unsupported file format'.format(filepath))
 
     return sft
+
+
+def filter_tractogram_data(tractogram, streamline_ids):
+    """ Filter tractogram according to streamline ids and keep
+    the data
+
+    Parameters:
+    -----------
+    tractogram: Tractogram or StatefulTractogram
```

Comment:
We have decided that we  would only deal with SFT from now on I think, no?

Reply:
See other conversation